devtools/maven/src/main/java/io/quarkus/maven/TrackConfigChangesMojo.java

```diff
@@ -75,7 +75,7 @@ public class TrackConfigChangesMojo extends QuarkusBootstrapMojo {
     /**
      * Dependency dump file
      */
-    @Parameter(property = "quarkus.track-config-changes.dependenciesFile")
```

Comment:
I suppose to be completely consistent, it would have to be `quarkus.trackConfigChanges.dependenciesFile`